Server/Components/Pawn/Scripting/Player/Natives.cpp

```diff
@@ -665,14 +665,14 @@ SCRIPT_API(GetPlayerSpecialAction, int(IPlayer& player))
 	return player.getAction();
 }
 
-SCRIPT_API_FAILRET(GetPlayerVehicleID, INVALID_VEHICLE_ID, int(IPlayerVehicleData& data))
+SCRIPT_API_FAILRET(GetPlayerVehicleID, 0, int(IPlayerVehicleData& data))
 {
 	IVehicle* vehicle = data.getVehicle();
 	if (vehicle)
 	{
 		return vehicle->getID();
 	}
-	return 0;
+	return FailRet;
```

Comment:
return INVALID_VEHICLE_ID here

Reply:
It's strange, but if you input an invalid player ID (a player who is not on the server) into GetPlayerVehicleID, it will return 65535 (INVALID_PLAYER_ID).

And if no player is in the vehicle, it will return 0.